SourceryTests/Models/TypedSpec.generated.swift

```diff
@@ -315,4 +315,4 @@ class TypedSpec: QuickSpec {
             }
         }
     }
-}
+}
```

Comment:
where is the lack of empty EOF line coming from? do you have some post-processing that happens outside of Sourcery?

Reply:
Sorry, my bad, i will be more careful in future. Cause of this, is that i run temporary version of sourcery by mistake inside repo. 

That was early version that trims end lines, that bug don't exist in current version.

i rechecked code, removed changes to package.resolved & regenerated SourceryRuntime code with sourcery binary.